dev/com.ibm.ws.jaxrs.2.0.security/src/com/ibm/ws/jaxrs20/appsecurity/security/JaxRsSSLManager.java

```diff
@@ -28,6 +29,9 @@
 public class JaxRsSSLManager {
     private static final TraceComponent tc = Tr.register(JaxRsSSLManager.class);
 
+    private static final Map<String, SSLSocketFactory> socketFactories = new HashMap<>();
+    private static SSLContext sslContext = null;
```

Comment:
Wouldn't we want this to be a Map too? (Map<String, SSLContext>) It seems like we might have more than one SSLContext configured for a given server, and if we just have one static field, then we might always skip the cache (i.e. app1 uses context1, then app2 uses context2 forcing a swap of the sslContext field, etc.).